elastic/kibana
# [Security Solution][Data Views] - Update bulk actions data views index param to be snake_case

## Summary

Overlooked a param name - should be snake case per our APIs. `overwriteDataViews` --> `overwrite_data_views`

### Testing
- Create 2 or more rules, one of which uses data views
- Select to bulk edit 2 or more rules, one of which uses data views
- Select to bulk add index, do not select any checkboxes and save --> rule with data view should remain using data view
- Select to bulk add index, select to overwrite rules with data views --> rule with data view should now indicate it is using added index pattern

<img width="481" alt="Screen Shot 2022-08-08 at 10 56 09 AM" src="https://user-images.githubusercontent.com/10927944/183484083-adb7e8f6-a2af-44e3-86be-e4e3450dc126.png">


### Checklist

    - In talks with docs team about new data views changes, this is part of the documentation

### For maintainers

    - This API param is new in 8.4
